src/database/postgres/insert_query_data.md

```diff
@@ -0,0 +1,49 @@
+## Insert and Query data
+
+[![postgres-badge]][postgres] [![cat-database-badge]][cat-database]
+
+The recipe inserts data into the `author` table. Then, displays the data from the `author` table.
+
+```rust,no_run
+extern crate postgres;
+
+use postgres::{Connection, TlsMode};
+
+struct Author {
+    id: i32,
+    name: String,
+    country: String
+}
+
+fn main() {
+    let conn = Connection::connect("postgresql://postgres:postgres@localhost/library", 
+                                    TlsMode::None).unwrap();
+    
+    let mut authors = HashMap::new();
+    authors.insert(String::from("Chinua Achebe"), "Nigeria");
+    authors.insert(String::from("Rabindranath Tagore"), "India");
+    authors.insert(String::from("Anita Nair"), "India");
+
+    for (key, value) in &authors {
+        let author = Author {
+            id: 0,
+            name: key.to_string(),
+            country: value.to_string()
+        };
+
+        conn.execute("INSERT INTO author (name, country) VALUES ($1, $2)",
+                 &[&author.name, &author.country]).unwrap();
+    }
+
+    // get all rows from author table
```

Comment:
[For the cookbook, we prefer descriptions over inline comments](https://github.com/rust-lang-nursery/rust-cookbook/pull/461#discussion_r222021790)